src/extension.ts

```diff
@@ -196,15 +200,25 @@ function convertToProviderStatus(crcStatus: string): extensionApi.ProviderStatus
     case 'Stopped':
       return 'stopped';
     case 'No Cluster':
-      return 'configured';
+      return 'stopped';
+    case 'Error':
+      return 'error';
     default:
       return 'not-installed';
   }
 }
 
 async function startStatusUpdateTimer(): Promise<void> {
+  if (statusFetchTimer) {
+    return; // we already set timer
+  }
   statusFetchTimer = setInterval(async () => {
     try {
+      // we don't need to update status while setup is going
+      if (isSetupGoing) {
```

Comment:
Maybe indicate that setup is going? WDYT?

Reply:
Podman desktop doesn't have such state. I
 add that, for case, when we perform `setup` before vm start, so user click `start` menu item, and we don't change extension state to `starting` until setup is not finished, which can teke a while, so that indicate that we are starting, even if actual start is not going yet.